foundationdb/README.md

```diff
@@ -0,0 +1,149 @@
+# Agent Check: FoundationDB
+
+## Overview
+
+This check monitors [FoundationDB][1] through the Datadog Agent. Aside from
+checking that the FoundationDB cluster is healthy, it also collects numerous metrics
+and, optionally, FoundationDB transaction logs.
+
+## Setup
+
+Both the check and the metrics apply to the FoundationDB cluster as a whole,
+and so should only be installed on one host (which doesn't need to be one that is
+running FoundationDB, but just one with access to it). The host chosen to
+collect the metrics is required to have the [FoundationDB client][8]
+installed. It is possible to monitor multiple FoundationDB clusters
+by adding multiple instances in the configuration file.
+
+Since logs are written on each machine, to collect logs, configure FoundationDB
+log collection on every host.
+
+Follow the instructions below to install and configure the check or log
+collection. For containerized environments, see the [Autodiscovery
+Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+To install the FoundationDB check on your host:
+
+1. If you do not already have the Datadog agent installed, [Download the
+   Datadog Agent][9].
+
+2. Install the [Datadog agent developer toolkit]
+(https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit)
+   on any machine.
+
+3. Run `ddev release build foundationdb` to build the package.
+
+4. Upload the build artifact to any host with an Agent and
+   run `datadog-agent integration install -w
+    path/to/foundationdb/dist/<ARTIFACT_NAME>.whl`.
+
+### Configuration
+
+### For service check and metrics
+
+Edit the `foundationdb.d/conf.yaml` file, in the `conf.d/` folder at
+the root of your Agent's configuration directory to start collecting
+FoundationDB metrics. See the [sample foundationdb.d/conf.yaml][3] for
+all available configuration options.
+
+The details of the cluster to check are found under the `instances`
+section. Some hints:
+
+* The `base_command` is correct by default if the `fdbcli` command is in
```

Comment:
Let's update to reflect the new implementation

Reply:
Done, and taken a pass over the README as a whole to see if anything else was needed.